test/pkg/environment/common/monitor.go

```diff
@@ -101,7 +101,7 @@ func (m *Monitor) NodeCountAtReset() int {
 
 // CreatedNodeCount returns the number of nodes created since the last reset
 func (m *Monitor) CreatedNodeCount() int {
```

Comment:
Doesn't this still fail since we don't have persistent tracking of all the nodes that we have created? I think CreatedNodes() still assumes there are no deletions

Reply:
Discussed offline. These two operations are different because one uses the raw data to determine which of the current set of nodes didn't exist at the reset while the other just blindly subtracts. We should favor using the raw data here.